PullRequestClosed/ReadMe.md

```diff
@@ -0,0 +1,57 @@
+# SubmitReview
+
+Move fixed Jira tickets using the "Merge into branch" or "Merge into master" transition after merging the pull request.
+
+If the pull request has been merged to a release branch like `master`, `main`, or `branch-*`, the `Merge into master` transition will be used. It would typically move a ticket from `APPROVED` to `CLOSED`. For any other target branch, a `Merge into branch` transition will be used.
+
+This action will attempt to move all tickets mentionned in the pull request title.
+
+## Inputs
+
+### `github-token`
+
+Token to access the GitHub API. 
+
+### `jira-user`
+
+User to access the Jira API.
+
+### `jira-token`
+
+Token to access the Jira API.
+
+## Outputs
+
+None
+
+## Example usage
+
+```yaml
+name: Close Pull Request
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  MergePullRequest_job:
+    name: Submit Review
+    runs-on: ubuntu-latest
+    # Single quotes must be used here https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals
+    # PRs from forks don't have required token authorization
+    if: |
+        github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.pull_request.merged
```

Comment:
We should align other examples to whether to include this if or not.
I did not include it in my examples.

Reply:
That's a good point. Please create an issue for it, so we don't forget about it. While thinking about this, the 

> # PRs from forks don't have required token authorization

might be actually a wrong statement, as we only read PR info now. Previously, we also did some write operations.

Reply:
#107